### PR TITLE
Set `startsAt` when Creating An Event without Changing Defaults

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5011

event-default-times

### CODE REVIEW

File: components/EditEventForm.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ import InputField from './InputField';
 import StyledButton from './StyledButton';
 import TimezonePicker from './TimezonePicker';
 
+const defaultStartsAt = new Date();
+defaultStartsAt.setHours(19);
+defaultStartsAt.setMinutes(0);
```
Comment (Betree) [inline components/EditEventForm.js:14]: I'm not a fan of having `defaultStartsAt` as a global variable, could we wrap that in a `getDefaultDate` helper?

**Review by Betree — APPROVED** [review]
Having that as part of the other PR would have been easier to review since we're changing the fields for `startsAt` end `endsAt` there. But that works for me if it solves the issue.

**Comment by znarf** [comment]
I don't see the point handling this in a separate PR.

**Comment by znarf** [comment]
Let's wait for the other to be merged and it will likely be conflicting.

**Comment by znarf** [comment]
This will be taken care elsewhere.

**Pull request closed without merging**